drivers/imx-enet-regs.h

```diff
@@ -314,7 +314,8 @@ typedef struct
 	/* first 3 same as enet_legacy_desc_t */
 	union {
 		struct {
-			uint16_t len, flags;
+			uint16_t len;
+			_Atomic volatile uint16_t flags;
```

Comment:
- this is read/written by the hardware, so changing this to atomic doesn't do the expected thing
- `volatile` is also not needed, as we're always declare the full struct as `volatile`